blogposts/2021/release-3.29.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Sourcegraph 3.29 release"
+publishDate: 2021-06-18T10:00-07:00
+description: "The Sourcegraph 3.29 release includes."
+tags: [blog, release]
+slug: "release/3.29"
+published: false
+changelogItems:
+
+---
+
+Sourcegraph 3.29 is now available! For this release, we added
+
+## Search results ranking
+Star count import from github.com/sourcegraph/ghdump is done. We now have 418321 repos with star data in the production repo table.
```

Comment:
We're making search ranking more intelligent, starting with better prioritization of repos vs. code in your search results. Search results and suggestions are now ranked by descending star count on sourcegraph.com and customer instances (for repos with stars), making it easier to find specific repos. Start searching on sourcegraph.com to experience the new ranking.